pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/ResourceQuota.java

```diff
@@ -18,11 +18,14 @@
  */
 package org.apache.pulsar.common.policies.data;
 
-import java.util.Objects;
+import lombok.Data;
+import lombok.experimental.Accessors;
 
 /**
  * Resource quota for a namespace or namespace bundle.
  */
+@Data
+@Accessors(chain = true)
```

Comment:
with `chain = true` we are introducing a breaking API change (changing the return type for the setters).
can we avoid to introduce this breaking change ?

Reply:
Good point. Will leave it as it was